src/components/RemediationActivityTable.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import {
+    Table,
+    TableHeader,
+    TableBody,
+    expandable
+} from '@patternfly/react-table';
+
+import { DateFormat } from '@redhat-cloud-services/frontend-components';
+
+import { StatusSummary, normalizeStatus } from './statusHelper';
+
+import { PermissionContext } from '../App';
+
+const RemediationActivityTable = ({ remediation, playbookRuns }) => {
+
+    const [ rows, setRows ] = useState([]);
+    const permission = useContext(PermissionContext);
+
+    const generateRows = (playbookRuns) => {
+        return (playbookRuns.reduce((acc, playbooks, i) => (
+            [
+                ...acc,
+                {
+                    isOpen: false,
+                    cells: [
+                        { title: <Link to={ `/${remediation.id}/${playbooks.id}` }><DateFormat type='exact' date={ playbooks.created_at }/></Link>,
+                            cellFormatters: [ expandable ]},
+                        `${playbooks.created_by.first_name} ${playbooks.created_by.last_name}`,
+                        { title: <StatusSummary
+                            executorStatus={ normalizeStatus(playbooks.status) }
+                            counts={ playbooks.executors.reduce((acc, ex) => (
+                                { pending: acc.pending + ex.counts.pending,
+                                    running: acc.running + ex.counts.running,
```

Comment:
Acked is missing. See all the possible statuses for an executor here: https://github.com/RedHatInsights/insights-remediations/blob/master/src/api/openapi.yaml#L625